src/main/java/com/cloudbees/plugins/credentials/cli/ListCredentialsAsXmlCommand.java

```diff
@@ -0,0 +1,62 @@
+package com.cloudbees.plugins.credentials.cli;
+
+import com.cloudbees.plugins.credentials.Credentials;
+import com.cloudbees.plugins.credentials.CredentialsProvider;
+import com.cloudbees.plugins.credentials.CredentialsStore;
+import com.cloudbees.plugins.credentials.CredentialsStoreAction;
+import com.cloudbees.plugins.credentials.cli.BaseCredentialsCLICommand;
+import com.cloudbees.plugins.credentials.domains.Domain;
+import com.cloudbees.plugins.credentials.domains.DomainCredentials;
+import hudson.Extension;
+import hudson.cli.CLICommand;
+import hudson.model.Items;
+import org.kohsuke.args4j.Argument;
+import org.kohsuke.args4j.Option;
+
+import java.io.OutputStreamWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+@Extension
+public class ListCredentialsAsXmlCommand extends BaseCredentialsCLICommand {
+
+    @Argument(metaVar = "STORE", usage = "Store ID. For example the Jenkins credentials global store would " +
+            "be identified by \"system::system::jenkins\"",
+            required = true)
+    private CredentialsStore store;
+
+    @Option(name = "--import", usage = "Read credentials as XML from the standard input and create them in the given store." +
+            "Input format is the same as the output generated by this command without enabling this option.")
+    private boolean doImport;
+
+
+    @Override
+    public String getShortDescription() {
+        return "Export and Import credentials as XML";
+    }
+
+    @Override
+    protected int run() throws Exception {
+        if (doImport) {
+            store.checkPermission(CredentialsProvider.UPDATE);
```

Comment:
should have `CREATE`, `UPDATE` (depending on my review comment below) and `MANAGE_DOMAINS` IIUC